config/01-ce-collector-audit-payload.conf

```diff
@@ -0,0 +1,24 @@
+# Settings for the collector audit payloads function.
+# The easiest way to direct the payload logging information here is to
+# set the environment variable CONDORCE_COLLECTOR_HOST on worker nodes
+# pointing to this condor-ce host.  Then pilot jobs that use condor
+# will automatically forward their payload classads here for auditing.
+
+# You can edit this file but that prevents future rpm upgrade
+# replacements, so it is better to override these settings in 
+# /etc/condor-ce/99-ce-collector-audit-payload.conf
```

Comment:
If it goes into `/etc/condor-ce/config.d`, then the admins should be able to edit it.  Defaults should go into `/usr/share/condor-ce/config.d` and _not_ be marked as config files.